collector/filesystem_linux.go

```diff
@@ -207,10 +209,11 @@ func parseFilesystemLabels(r io.Reader) ([]filesystemLabels, error) {
 		parts[1] = strings.Replace(parts[1], "\\011", "\t", -1)
 
 		filesystems = append(filesystems, filesystemLabels{
-			device:     parts[0],
-			mountPoint: rootfsStripPrefix(parts[1]),
-			fsType:     parts[2],
-			options:    parts[3],
+			device:       parts[0],
+			mountPoint:   rootfsStripPrefix(parts[1]),
+			fsType:       parts[2],
+			options:      parts[3],
+			device_error: "",
```

Comment:
This should be `deviceError` to match Go naming conventions. Odd that our linter isn't picking this up.